packages/terra-tabs/src/Tabs.jsx

```diff
@@ -138,7 +138,7 @@ class Tabs extends React.Component {
     return (
       <CommonTabs
         id={customProps.id}
-        activeItemKey={this.state.activeKey}
+        activeItemKey={this.props.activeKey || this.state.activeKey}
```

Comment:
we can make use of previously initialised activeKey variable instead of this condition

Reply:
Done https://github.com/cerner/terra-framework/pull/1745/commits/9491ceeefbb8edcd28558754f97d71b9291318ec